src/ui/components/tusktask/group/TaskCard.tsx

```
@@ -0,0 +1,165 @@
+                          subTask.subTasks.filter((task) => task.completedAt)
+                            .length
+                        }
+                        icon={ClockAlert}
+                      />
+                      <StatusOverviewCard
+                        title="Today"
+                        number={
+                          subTask.subTasks &&
+                          subTask.subTasks.filter(
+                            (task) => task.status !== "completed"
+                          ).length
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)